client/signup/steps/user/index.jsx

```diff
@@ -367,6 +385,10 @@ export class UserStep extends Component {
 		return this.props.step && 'completed' === this.props.step.status;
 	}
 
+	userCreationCompletedWithAndHasHistory( props ) {
```

Comment:
Would this be clearer without the `With`?
Am I interpreting this identifier correctly if I insert a pause like this? "user creation completed, and has history"

Reply:
good call